guacamole/src/main/frontend/src/app/settings/styles/history-player.css

```diff
@@ -96,23 +96,15 @@
     background: rgba(0, 0, 0, 0.5);
 }
 
-.settings.connectionHistoryPlayer .guac-player-controls.playing {
+.settings.connectionHistoryPlayer .guac-player-controls {
     opacity: 0;
     -webkit-transition: opacity 0.25s linear 0.25s;
     -moz-transition: opacity 0.25s linear 0.25s;
     -o-transition: opacity 0.25s linear 0.25s;
     transition: opacity 0.25s linear 0.25s;
 }
 
-.settings.connectionHistoryPlayer .guac-player-controls.paused,
-.settings.connectionHistoryPlayer .guac-player-controls.playing:hover {
-    opacity: 1;
-    -webkit-transition-delay: 0s;
```

Comment:
The default value for these properties should be 0 anyway, so I don't think we need these.